### PR TITLE
Define views for district and service, Add error response helper

### PR DESCRIPTION
**Pull request by Pavi114**
Yet to add location and editing services. Will do so after merging @AananthV 's pr.

### CODE REVIEW

File: api/views/service.py [hunk 1]
```diff
@@ -26,13 +36,37 @@ def put(self, request):
...
+        return {}
 
     def post(self, request):
         '''
         Edit a service
         Parameters:
-            body with service info
+            body with service info, serviceId of the service to be changed
         '''
         return { status: 200, message: "success" }
```
Comment (KIRA009) [inline api/views/service.py]: javascript vibes

File: api/views/service.py [hunk 2]
```diff
@@ -42,4 +76,13 @@ def delete(self, request):
         Parameters:
             body with service info
         '''
-        return { status: 200, message: "success" }
\ No newline at end of file
+        service_id = request.POST.get('service_id')
+        try:
+            Service.objects.get(id=service_id).delete()
+            return {
+                status: 200,
```
Comment (KIRA009) [inline api/views/service.py:155]: same here too

**Review by KIRA009 — CHANGES_REQUESTED** [review]
Few more pep8 issues which would light up my pycharm, but they aren't breaking issues, so can deal with them later

File: api/views/service.py [hunk 3]
```diff
@@ -26,20 +37,124 @@ def put(self, request):
...
+                    )
+            
+            if not is_active:
+                ServiceChangeRequest.objects.create(
+                    user=user,
+                    changes=service
+                )
+            
+            return {
+                status: 200,
```
Comment (KIRA009) [inline api/views/service.py]: javscript vibes

File: api/views/service.py [hunk 4]
```diff
@@ -26,20 +37,124 @@ def put(self, request):
...
+                    user=user,
+                    changes=new_service,
+                    service=service
+                )
+            else:
+                service.is_active = False
+                service.save()
+
+            return {
+                status: 200,
```
Comment (KIRA009) [inline api/views/service.py]: here too

File: api/views/service.py (hunk 3 above)
Comment (Pavi114) [reply api/views/service.py]: Gg will remove

**Pull request merged**